ChessBoard.Api/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+services:
+  mongo:
+    image: mongo
+    container_name: mongo
+    ports:
+      - "27017:27017"
+    volumes:
+      - mongodbdata:/data/db
+    networks:
+      - chessboard-network
+
+  chessboard:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: chessboardapi:v1.0
+    container_name: chessboardapi
+    ports:
+      - "8080:80"
```

Comment:
Good answer, no need to dig deeper for now, we'll get to that at some point.

In theory if you don't have any application running locally that uses port 80 it would not be a problem to run it on 80, that way you could simply skip it in your url `http://localhost/swagger/index.html` for local testing etc. etc. but for now it's perfectly fine to have it on 8080